options.go

```diff
@@ -115,5 +120,6 @@ func optionsWithDefaults(opts Options) Options {
 	}
 	opts.SessionName = defaults.String(opts.SessionName, "_buffalo_session")
 	opts.Host = defaults.String(opts.Host, envy.Get("HOST", fmt.Sprintf("http://127.0.0.1:%s", envy.Get("PORT", "3000"))))
+	opts.ShutDownTimeout = 30 * time.Second
```

Comment:
This will overwrite any setting a user has set. you should check whether it has been set already or not.

Reply:
Doh.... I'll open a PR for the `going` helpers to keep the default checks together